File: TransCoder_tokenized_test_set_functions/cpp/FIND_UNIQUE_ELEMENTS_MATRIX.cpp
int unique ( int mat [ R ] [ C ], int n, int m ) {
  int maximum = 0, flag = 0;
  for ( int i = 0;
  i < n;
  i ++ ) for ( int j = 0;
  j < m;
  j ++ ) if ( maximum < mat [ i ] [ j ] ) maximum = mat [ i ] [ j ];
  int b [ maximum + 1 ] = {
    0 };
    for ( int i = 0;
    i < n;
    i ++ ) for ( int j = 0;
    j < m;
    j ++ ) b [ mat [ i ] [ j ] ] ++;
    for ( int i = 1;
    i <= maximum;
    i ++ ) if ( b [ i ] == 1 ) cout << i << " ";
    flag = 1;
    if ( ! flag ) {
      cout << "No unique element in the matrix";
    }
  }

File: TransCoder_tokenized_test_set_functions/python/FIND_UNIQUE_ELEMENTS_MATRIX.py
def unique ( mat, n, m ) :
    maximum = 0;
    flag = 0
    for i in range ( 0, n ) :
        for j in range ( 0, m ) :
            if ( maximum < mat [ i ] [ j ] ) :
                maximum = mat [ i ] [ j ];
                
    uniqueElementDict = [ 0 ] * ( maximum + 1 )
    for i in range ( 0, n ) :
        for j in range ( 0, m ) :
            uniqueElementDict [ mat [ i ] [ j ] ] += 1
    for key in range ( maximum + 1 ) :
        if uniqueElementDict [ key ] == 1 :
            print ( key, end = " " )
            flag = 1
    if ( flag == 0 ) :
        print ( "No unique element in the matrix" )
    
